### DISCUSSION TITLE
Geometry Output from Meshinstancer

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#140] [Feature Requests]
Hi, was looking into fractals/recursive geometry - I made a test patch which kinda works but not exactly https://cables.gl/edit/DowqnW (follow the comment inside the patch - doesnt work if you save it with the value) 
So was wondering if it might be possible to have a geometry object output for meshinstancer - it will basically allow meshinstancing a meshinstancer and have some sort of controlled recursion.

**Comment by aagxd** [comment]
Other day wanted to combine instancer and spline deform, this would save me a lot of trouble!

**Comment by steam0r** [comment]
didnt we try to fix this in meetup and found a way to make this work with less tiny cubes?

**Reply by franckaubry** [reply]
Oooh! Really? Please share this technique :)
